tests/unittests/OperatorTest.cpp

```diff
@@ -8516,6 +8516,64 @@ TEST_P(OperatorTest, SigmoidOverflow) {
   }
 }
 
+/// This unit test exposes a problem with the CPU Sigmoid when stacking a higher
+/// number of operations for extreme input values which result in NaNs.
+TEST_P(OperatorTest, SigmoidOverflowCPUStacking) {
+  ENABLED_BACKENDS("CPU");
```

Comment:
nit: Can you use `CHECK_IF_ENABLED()` here and the other test? I imagine this sort of extreme value issue could occur on other CPU-like architectures so worth getting the coverage.

Reply:
Done.